hbase-shell/src/main/ruby/shell/commands/snapshot.rb

```diff
@@ -24,7 +24,7 @@ def help
 Take a snapshot of specified table. Examples:
 
   hbase> snapshot 'sourceTable', 'snapshotName'
-  hbase> snapshot 'namespace:sourceTable', 'snapshotName', {SKIP_FLUSH => true}
+  hbase> snapshot 'namespace:sourceTable', 'snapshotName', {SKIP_FLUSH => true | MAX_FILESIZE => 21474836480}
```

Comment:
Shouldn't it be separated with comma instead of pipe? `{SKIP_FLUSH => true, MAX_FILESIZE => 21474836480}`

Reply:
Done.